gradio/deprecation.py

```diff
@@ -38,6 +38,8 @@ def check_deprecated_parameters(cls: str, **kwargs) -> None:
             kwargs.pop(key)
             # Interestingly, using DeprecationWarning causes warning to not appear.
             warnings.warn(value)
+    if "_api_mode" in kwargs:
```

Comment:
So that we do not get warnings about un-used kwargs. I also don't want to use `api_mode` since that would raise a warning about `api_mode` being deprecated so I call the parameter `_api_mode`.